web/docs/faqs.md

```diff
@@ -49,7 +49,7 @@ access the platform. All computation and storage takes place at your nodes. The
 platform acts as rendezvous point that connects two TLS-encrypted channels, one
 from the node to the platform, and one from the browser to the platform:
 
-![Platform Connections](platform-connections.excalidraw.svg)
+![Platform Connections](platform-connections.svg)
```

Comment:
Is it intended that this one no longer embeds the Excalidraw scene?

Reply:
Yes, this makes the images a lot smaller and I don't think we have a lot of ROI from embedding the source. Historically, we haven't had a single contribution that relied on this (which was the primary reason to do it in the first place).